CommonControls/FileTypes/RigidModel/MaterialHeaders/WeightedMaterial.cs

```diff
@@ -396,7 +427,10 @@ List<RmvVector4> LoadVec4Params(uint Vec4ParamCount, byte[] dataArray, ref int d
 
     }
 
-    struct WeightedMaterialStruct
+    /// <summary>
+    /// TODO: Renamed this struct from "WeightedMaterialStruct" as it the same for several RigidMaterials
+    /// </summary>
+    struct CommonMaterialStruct
```

Comment:
Why is this needed?

Reply:
While its true that this is shared, the code is not written in such a way that it is. Hence it should keep its old name. 
If the code is changed in the future (and different PR) the struct can be renamed